xds/src/main/java/io/grpc/xds/orca/OrcaOobUtil.java

```diff
@@ -569,6 +548,11 @@ public void start(SubchannelStateListener listener) {
       orcaState.init(this, listener);
```

Comment:
We need to _just_ do `super.start(listener)` if this is a reused orcaState. (Don't call init())